Source/Plugins/Core/com.equella.core/resources/web/sass/legacy.scss

```
@@ -374,7 +374,7 @@ center,
 dl,
 dt,
 dd,
-ol,
+ol:not(#DRM-terms),
```

Is there a reason not go with the generic solution on `iframe` above? That is, could this be:

```suggestion
ol:not(.skip-css-reset),
```

---

I will use this generic solution :+1: